torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -638,6 +638,40 @@ def rewrite_node(node):
                     input = g.call_function(ttnn.to_layout, args=(input, TtnnRowMajorLayout()))
                 return g.call_function(ttnn.pad, args=(input, full_pad, value))
 
+            if node.target == torch.ops.aten.convolution.default:
+                output_shape = node.meta["val"].size()
+                input_shape = args[0].meta["val"].size()
+                weight_shape = args[1].meta["val"].size()
+
+                if len(input_shape) == 4:
+                    in_n, in_c, in_h, in_w = input_shape
+                    out_n, out_c, out_h, out_w = output_shape
+                    input_nhwc = g.call_function(ttnn.permute, (args[0], (0, 2, 3, 1)))
```

Comment:
This patch also fails.
```suggestion
                    input_nhwc = g.call_function(ttnn.permute, (args[0], (0, 2, 3, 1)))
                    input_nhwc = g.call_function(ttnn.to_layout, (input_nhwc, TtnnRowMajorLayout()))
```